frontend-mini-challenge/components/Alerts.tsx

```diff
@@ -17,12 +22,10 @@ export function ErrorAlert({ children, title }) {
   );
 }
 
-export function InfoAlert({ children, title, Icon }) {
-  Icon = Icon || InfoIcon;
```

Comment:
This line is here on purpose. 

If Icon is passed in then use that, otherwise use InfoIcon as the default. 

Please don't remove functionality. Feel free to refactor, but don't just delete code.